irohad/main/irohad.cpp

```diff
@@ -130,6 +132,7 @@ int main(int argc, char *argv[]) {
   // Configuring iroha daemon
   Irohad irohad(config[mbr::BlockStorePath].GetString(),
                 config[mbr::PgOpt].GetString(),
+                kListenIP, // TODO add a param in config?
```

Comment:
We have a special TODO format, which should contain:
- author (nick-)name
- date, when todo was written
- task id in jira
- (optionally) some commend

E.g: `TODO(@l4l) 13/10/18: make something at IR-1234`

Reply:
done